aten/src/ATen/native/mkldnn/xpu/Conv.cpp

```diff
@@ -737,12 +813,165 @@ std::tuple<Tensor, Tensor, Tensor> convolution_backward_overrideable(
   return std::tuple<Tensor, Tensor, Tensor>{grad_input, grad_weight, grad_bias};
 }
 
+Tensor convolution_pointwise(
+    const Tensor& input_t,
+    const Tensor& weight_t,
+    const c10::optional<Tensor>& bias_opt,
+    IntArrayRef padding,
+    IntArrayRef stride,
+    IntArrayRef dilation,
+    int64_t groups,
+    c10::string_view attr,
+    torch::List<c10::optional<at::Scalar>> scalars,
+    c10::optional<c10::string_view> algorithm) {
+  Attr att;
+  att = construct_unary_attr(attr, scalars, algorithm, att);
+  const Tensor bias = bias_opt.has_value() ? bias_opt.value() : at::Tensor();
+  std::vector<int64_t> output_padding = {0};
+
+  return _convolution(
+      input_t,
+      weight_t,
+      bias,
+      stride,
+      padding,
+      dilation,
+      /*transposed*/ false,
+      output_padding,
+      groups,
+      att);
+}
+
+Tensor convolution_pointwise_binary(
+    const Tensor& input_t,
+    const Tensor& other_t,
+    const Tensor& weight_t,
+    const c10::optional<Tensor>& bias_opt,
+    IntArrayRef padding,
+    IntArrayRef stride,
+    IntArrayRef dilation,
+    int64_t groups,
+    c10::string_view binary_attr,
+    c10::optional<at::Scalar> alpha,
+    c10::optional<c10::string_view> unary_attr,
+    torch::List<c10::optional<at::Scalar>> unary_scalars,
+    c10::optional<c10::string_view> unary_algorithm) {
+  Tensor output;
+  Tensor bias = bias_opt.has_value() ? bias_opt.value() : at::Tensor();
+  // Step1: Construct binary attr
+  Attr attr;
+  bool is_fused = true;
+  if (binary_attr != "add") {
+    attr = construct_binary_attr(binary_attr, alpha, other_t, attr);
+  } else {
+    attr = get_onednn_conv_sum_attr(
+        input_t,
+        weight_t,
+        stride,
+        padding,
+        dilation,
+        other_t,
+        alpha.has_value() ? alpha.value().toFloat() : 1.f,
+        output,
+        /*is_fused*/ is_fused,
+        attr);
+  }
+
+  // Step2: Append unary attr
+  if (unary_attr.has_value())
+    attr = construct_unary_attr(
+        unary_attr.value(), unary_scalars, unary_algorithm, attr);
+
+  Tensor res = _convolution_out(
+      output,
+      input_t,
+      weight_t,
+      bias,
+      stride,
+      padding,
+      dilation,
+      /*transpoced*/ false,
+      {{0, 0}},
```

Comment:
The output_padding `{{0, 0}}` only for conv2d, conv3d will get error. Make it only one dim to support nD conv
```suggestion
      {{0}},
```